generators/upgrade/index.js

```diff
@@ -214,7 +224,7 @@ module.exports = UpgradeGenerator.extend({
         updateJhipster: function() {
             this.log(chalk.yellow('Updating ' + GENERATOR_JHIPSTER + '. This might take some time...'));
             var done = this.async();
-            shelljs.exec('npm install -g ' + GENERATOR_JHIPSTER, {silent:true}, function (code, msg, err) {
+            shelljs.exec('npm install ' + GENERATOR_JHIPSTER, {silent:true}, function (code, msg, err) {
```

Comment:
not sure if it should be only installed locally. Why do you think it can fail if installed globally?